### PR TITLE
MS SQL in Database,Visual Studio in Software and SVN in Devops added.

### PR DESCRIPTION
**Pull request by umaad**
I have added mssql in database,visual studio in software and svn in devops.

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -138,6 +138,9 @@ const icons = {
     framer: 'https://www.vectorlogo.zone/logos/framer/framer-icon.svg',
     bulma: 'https://raw.githubusercontent.com/gilbarbara/logos/804dc257b59e144eaca5bc6ffd16949752c6f789/logos/bulma.svg',
     couchdb: 'https://raw.githubusercontent.com/devicons/devicon/0d6c64dbbf311879f7d563bfc3ccf559f9ed111c/icons/couchdb/couchdb-original.svg',
+	visualstudio: 'https://devicons.github.io/devicon/devicon.git/icons/visualstudio/visualstudio-plain.svg',
+    mssql:'https://www.brandeps.com/logo-download/M/Microsoft-sql-server-logo-vector-01.svg',
+    svn:'https://logodix.com/logo/1247596.png',
```
Comment (rahuldkjain) [inline src/constants/skills.js:143]: please use svg images only

File: src/constants/skills.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const skills = ['vuejs', 'react', 'angularjs', 'aws', 'android', 'backbonejs', '
     'nodejs', 'python', 'swift', 'spring', 'oracle', 'nginx', 'linux', 'redux', 'webpack', 'express', 'flutter', 'dart', 'kotlin',
     'tensorflow', 'chartjs', 'jenkins', 'gcp', 'kubernetes', 'azure', 'git', 'kafka', 'solr', 'cassandra', 'rabbitMQ',
     'hadoop', 'bash', 'pytorch', 'opencv', 'illustrator', 'photoshop', 'figma', 'blender', 'babel', 'sketch',
-    'xamarin', 'vagrant', 'tailwind', 'materialize', 'invision', 'framer', 'couchdb', 'bulma' ];
+    'xamarin', 'vagrant', 'tailwind', 'materialize', 'invision', 'framer', 'couchdb', 'bulma','visualstudio','msql','svn' ];
```
Comment (rahuldkjain) [inline src/constants/skills.js:8]: typo here `msql` instead of `mssql`. Please check

Comment (rahuldkjain) [reply src/constants/skills.js:8]: Visual Studio is an IDE and I think there is no need to show IDE on github profile. So I suggest you to please remove `Visual Studio`

**Comment by harshcut** [comment]
Closing the Pull Request due to inactivity.
Is this a mistake? Open a new Pull Request linking this PR for reference.

**Pull request closed without merging**